spec/models/resource_action_serializer_spec.rb

```diff
@@ -4,14 +4,16 @@
   describe "#serialize" do
     let(:resource_action) do
       ResourceAction.new(
-        :dialog_id     => 123,
-        :resource_id   => 321,
-        :created_at    => Time.now,
-        :updated_at    => Time.now,
-        :resource_type => "DialogField",
-        :ae_namespace  => "Customer/Sample",
-        :ae_class      => "Methods",
-        :ae_instance   => "Testing"
+        :dialog_id                   => 123,
+        :resource_id                 => 321,
+        :created_at                  => Time.now,
+        :updated_at                  => Time.now,
+        :resource_type               => "DialogField",
+        :ae_namespace                => "Customer/Sample",
+        :ae_class                    => "Methods",
+        :ae_instance                 => "Testing",
+        :configuration_template_id   => 333,
+        :configuration_template_type => "ConfigurationScript"
```

Comment:
Hardcoded ids and types immediately scare me because they don't take into account regions as well as changes to the base class.  This should be done with FactoryGirl.build and attaching to "real" resources which should also be FactoryGirl built.  So for example, your configuration_template_type here is wrong because it will actually be ConfigurationScriptBase.  By hardcoding you don't see that problem until it's too late.

I know it's already in this PR, but I think this should be fixed before merged, personally.